code/modules/mining/lavaland/loot/legion_loot.dm

```diff
@@ -8,16 +8,22 @@
 	w_class = WEIGHT_CLASS_BULKY
 	force = 25
 	damtype = BURN
-	hitsound = 'sound/weapons/sear.ogg'
+	resistance_flags = LAVA_PROOF | FIRE_PROOF | ACID_PROOF
 	needs_permit = TRUE
-	var/storm_type = /datum/weather/ash_storm
-	var/storm_cooldown = 0
+	var/max_thunder_charges = 3
+	var/thunder_charges = 3
+	var/thunder_charge_time = 15 SECONDS
+	var/static/list/excluded_areas = list(/area/space)
+	var/list/targeted_turfs = list()
```

Comment:
Might want to make sure you clear this list on destroy, and possibly autodoc it

Reply:
done!